faucet/src/http.rs

```diff
@@ -532,6 +601,8 @@ mod tests {
         let user_allowance_dot: u128 = 1;
         let vault_allowance_dot: u128 = 500;
         let staked_relayer_allowance_dot: u128 = 500;
+        let one_dot: u128 = 10_000_000_000;
```

Comment:
```suggestion
        let one_dot: u128 = 10 ** 10;
```

Reply:
That doesn't seem to be valid syntax, I used `10u128.pow(10)`